app/controllers/calls_controller.rb

```diff
@@ -88,7 +88,12 @@ def render_twiml(response)
 
   def active_call
     remote_id = params['CallSid'] || params[:remote_id]
-    @call = Ivr::Call.includes(connections: :legislator).where(remote_id: remote_id).first_or_create
+    @call = Ivr::Call.includes(connections: :legislator).first_or_initialize(remote_id: remote_id)
+    if @call.new_record?
+      @call.person ||= Person.find_or_create_by(phone: params[:From])
```

Comment:
Yeah, I still want to capture potential phone numbers that might come in if people call back later.  I'm thinkign the most likley case is someone gets called.  Finished and wants to call back.  Or they don't pick up and try to call back.  In both of those cases an email sould exist, but just in case some random tries to call the line, I want to capture their data.